anserini_dependency/README.md

```diff
@@ -34,22 +31,13 @@ Install the dependency packages:
 
 ```
 cd Castor
-pip3 install -r requirements.txt
+pip install -r requirements.txt
```

Comment:
Do all our models support both py2 and py3?

Reply:
No, it only supports Python 3. But if they followed the setup correctly `pip` should be the correct one by default - if they explicitly use `pip3` and `pip` doesn't work something is wrong with their environment set-up.